SpringProject/src/main/java/com/gradproject2019/auth/service/AuthServiceImpl.java

```diff
@@ -53,9 +53,10 @@ public void logout(UUID token) {
         authRepository.deleteById(token);
     }
 
-    private void checkTokenExists(UUID token) {
+    public boolean checkTokenExists(UUID token) {
```

Comment:
the naming convention for methods returning boolean is for them to start with is. For example isExistingToken

Reply:
Does this need to return a boolean though